src/components/tooltip/NetworkMarker.js

```diff
@@ -76,20 +76,27 @@ function _markerDetailBytesNullable(label: string, value: ?number): * {
   );
 }
 
-const ALL_NETWORK_PROPERTIES_IN_ORDER = [
-  'startTime',
+const PRECONNECT_PROPERTIES_IN_ORDER = [
   'domainLookupStart',
   'domainLookupEnd',
   'connectStart',
   'tcpConnectEnd',
   'secureConnectionStart',
   'connectEnd',
+];
+
+const ALL_NETWORK_PROPERTIES_IN_ORDER = [
+  'startTime',
+  ...PRECONNECT_PROPERTIES_IN_ORDER,
```

Comment:
I decided this so that I wouldn't repeat this, I was afraid of typo. But tell me if you prefer that I repeat these.

Reply:
This is fine.